### ISSUE TITLE
Success 

### ISSUE THREAD

**Issue by trevor-clarke** [allepicondor/BestBuyScalperBot#1]
Hey! I was just wondering if you have had any success with this bot yet?

**Comment by allepicondor** [comment]
Yes it works great for any product on Best Buy response time is sub 10 seconds to when the product is available
